config/puma/production.rb

```diff
@@ -27,6 +27,9 @@
 #
 preload_app!
 
+bind ENV.fetch('FOREMAN_BIND', '127.0.0.1')
+port ENV.fetch('FOREMAN_PORT', '3000')
```

Comment:
due to https://github.com/theforeman/puppet-foreman/pull/883#discussion_r485405549 I went and start reading puma code (welp, what else do you do at 10 in the morning?) and I think this line will break things.

see https://github.com/puma/puma/blob/v4.3.3/lib/puma/dsl.rb#L191-L194

`port <port>` will internally generate a `bind tcp://<host>:<port>` so the above becomes:

```
bind <FOREMAN_BIND/127.0.0.1>
bind tcp://<host>:<FOREMAN_PORT/3000>
```

1. `bind 127.0.0.1` is invalid as it had no schema (and I don't think puma defaults to `tcp://` in this case) and no port
2. the second `bind` will open up a tcp socket, most probably on `0.0.0.0` which is exactly what we want to avoid with this dance.